src/app/lista-nominal/ListConteiner.tsx

```diff
@@ -0,0 +1,306 @@
+            )
+        },
+    },
+    {
+        field: 'prazo_proxima_afericao_pa',
+        headerName: 'Prazo para próxima aferição de PA',
+        width: 200 ,
+        headerAlign: 'left',
+        align: 'left',
+        renderCell({ value }) {
+            const theme = value === "Em dia" ? "success" : "warning";
+            return(
+                <TableTag
+                    theme={theme}
+                    text={value}
+                    icon={{
+                        src: IconDetailsMap[theme].src,
+                        alt: IconDetailsMap[theme].alt
+                    }}
+                />
+            )
+        },
+    },
+    {
+        field: 'acs_nome_cadastro',
+        headerName: 'ACS responsável',
+        width: 250 ,
+        headerAlign: 'left',
+        align: 'left'
+    },
+] as GridColDef[];
+
+interface Filter {
+    id: string;
+    label: string;
+    options: { value: string; label: string }[];
+    multiSelect: boolean;
+    width: string;
+}
+
+type ListDataTotalRows = number;
+type ListData = {
+    data: Record<string, string | number | Date>[];
+    totalRows: ListDataTotalRows;
+};
+// Adicionar união de valores quando soubermos as listas que teremos
+interface ListConteinerProps {
+    list: string;
+}
+
+export const ListConteiner = ({ list }: ListConteinerProps) => {
+    const { data: session } = useSession();
+    const [value, setValue] = useState<Record<string, string | string[]>>({
+        filter1 : [],
+        filter2 : [],
+        filter3 : "",
+    });
+    const [tableData, setTableData] = useState<ListData>({
+        data: [],
+        totalRows: 0,
+    });
+    const [pagination, setPagination] = useState<GridPaginationModel>({
+        page: 0,
+        pageSize: 8,
+    });
+    const [sorting, setSorting] = useState<GridSortModel>([{
+        field: 'nome',
+        sort: 'asc'
+    }]);
+    const [isLoading, setIsLoading] = useState(false);
+
+    useEffect(() => {
+        const fetchData = async () => {
+            setIsLoading(true);
+            setTableData({
+                data: [],
+                totalRows: 0,
+            });
+            setIsLoading(false);
+        };
+
+        if (session && session.user) fetchData();
+    }, [pagination, sorting, session, list]);
+
+    function handleSortModelChange(newSortModel: GridSortModel) {
+        setSorting([...newSortModel]);
+    }
+
+    const clearFiltersArgs = {
+        iconActive : "https://media.graphassets.com/1EOGJH6TvSMqTrjigY1g",
+        iconInactive : "https://media.graphassets.com/x37RkcUrTH6G50ganj9d",
+        label: 'Limpar todos os filtros',
+    }
+
+    const filters = ManyFiltersData.map((filter: Filter) => (
+        <SelectDropdown 
+            key={filter.id} 
+            {...filter} 
+            value={value} 
+            setValue={setValue} 
+            options={filter.options} 
+            label={filter.label} 
+            multiSelect={filter.multiSelect} 
```

Comment:
O PR do dan eu ja propus essa alteração
 
```suggestion
            isMultiSelect={filter.isMultiSelect} 
```

Reply:
Beleza, puxo da branch dele então